src/app/exams/exams-question.component.html

```diff
@@ -33,19 +33,26 @@
       </mat-form-field>
       <button type="button" mat-button (click)="addChoice()" class="margin-lr-5" *ngIf="questionForm.controls.type.value === 'select'">Add Choice</button>
     </div>
-    <div class="question-choices">
+    <div *ngIf="examType !== 'surveys'" class="exam-question-choices">
```

Comment:
@leahphuong you can just use conditional class name instead of having *ngIf and repeated code block